src/main/resources/application.yml

```diff
@@ -30,4 +30,7 @@ kafka:
   brokerConnect: localhost:9092
   isSecured: false
   saslMechanism: "PLAIN"
-  securityProtocol: "SASL_PLAINTEXT"
\ No newline at end of file
+  securityProtocol: "SASL_PLAINTEXT"
+  truststoreLocation : "${TRUSTSTORE_LOCATION:/kafka.truststore.jks}"
```

Comment:
You have `_LOCATION` at the end of the environment variable, whereas in the `README.md` and in `kafdrop.sh` there is no `_LOCATION` suffix at the end of this environment variable. Is this right?

Reply:
Oh yes, my bad. I check that ;)